.changelog/unreleased/bug-fixes/3992-light-panic.md

```diff
@@ -0,0 +1,4 @@
+- `[light]` Print an error if starting from the trusted store, but it's empty
+  (due to incorrect configuration or user forgetting to provide a trusted
```

Comment:
```suggestion
  This can happen using the `light` CometBFT command-line command while using a fresh trusted store and no trusted height and hash are provided.
```